### PR TITLE
[10.0][sale_cancel_reason]Added description field

### PR DESCRIPTION
**Pull request by paolomonniLP**
Added a text field for the description of the cancellation of the order.

### CODE REVIEW

File: sale_cancel_reason/wizard/cancel_reason.py [hunk 1]
```diff
@@ -14,6 +14,7 @@ class SaleOrderCancel(models.TransientModel):
         'sale.order.cancel.reason',
         string='Reason',
         required=True)
+    description = fields.Text(string='Description', size=12)
```
Comment (pedrobaeza) [inline sale_cancel_reason/wizard/cancel_reason.py]: You put a text field but then limited to 12 characters. Isn't this a nonsense?

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
What is the use case for 2 strings for the reason?

**Comment by paolomonniLP** [comment]
This field is used to describe in detail the reason for canceling the order.

**Comment by pedrobaeza** [comment]
Uhm, I think you don't need too much explanations about cancel reasons (they are internal), but I let other reviewers to decide.

**Comment by JayVora-SerpentCS** [comment]
Agree with @pedrobaeza .Simple M2O is enough. Always minimise fields. If the reason is different, it should be mentioned in a comment may be. Or choose OTHER as a reason on sale.order.cancel.reason

File: sale_cancel_reason/model/sale.py [hunk 2]
```diff
@@ -7,12 +7,10 @@
 class SaleOrder(models.Model):
     _inherit = 'sale.order'
 
-    cancel_reason_id = fields.Many2one(
-        'sale.order.cancel.reason',
-        string="Reason for cancellation",
-        readonly=True,
+    cancel_reason_id = fields.Many2one('sale.order.cancel.reason', string="Reason for cancellation", readonly=True,
```
Comment (JayVora-SerpentCS) [inline sale_cancel_reason/model/sale.py:10]: You made travis go red forcefully:) Was right earlier.

**Comment by JayVora-SerpentCS** [comment]
ping @paolomonniLP

**Comment by rousseldenis** [comment]
Closing this as very old and comments not attended. Please feel free to reopen if needed.

**Pull request closed without merging**